app/Console/ConcludeCommand.php

```diff
@@ -76,34 +76,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
         return 0;
     }
 
-    private function createSolrQuery(Carbon $lowerDateBoundary = null, Carbon $upperDateBoundary = null)
+    private function createLuceneQuery(Carbon $lowerDateBoundary, Carbon $upperDateBoundary): string
     {
-        $sapiDateRange = $this->createSolrDateRangeString($lowerDateBoundary, $upperDateBoundary);
+        $sapiDateRange = $this->createLuceneDateRangeString($lowerDateBoundary, $upperDateBoundary);
 
-        return "type:event AND availableto:{$sapiDateRange}";
+        return "_type:event AND availableRange:{$sapiDateRange}";
```

Comment:
Nit: Might be a bit easier to read this method if we inline the `createLuceneDateRangeString()` logic? Since it's only called once anyway.

```
private function createLuceneQuery(Carbon $lowerDateBoundary, Carbon $upperDateBoundary): string
{
    $from = $lowerDateBoundary ? $this->getLuceneFormattedDateTime($lowerDateBoundary) : '*';
    $to = $this->getLuceneFormattedDateTime($upperDateBoundary);

    return "_type:event AND availableRange:[{$from} TO {$to}]";
}
```

Reply:
Good point! Even the `getLuceneFormattedDateTime` has little added value.